### ISSUE TITLE
kyverno-flux - new k8s bosh deployment from scratch failure

### ISSUE THREAD

**Issue by poblin-orange** [orange-cloudfoundry/paas-templates#1524]
### Expected behavior

* As a paas-templates operator 
* In order to 
  * have a clean end to end k3s cluster creation (bosh deploy , inducing kyverno, fluxcd install then bridge Kustomization loading all k8s artifact loading) 
* I need 

### Observed behavior
- Kyverno helm chart   kyverno-v2.0.3 (app v1.4.3 )
 
- Kyverno policy fails to apply on fluxcd, resuling in empty Kustomization (no spec structure, no CRD validation failure by flux)

```
 kyverno E0211 15:22:20.599167       1 mutation.go:87] WebhookServer "msg"="mutate error" "error"="failed to validate resource mutated by policy root-kustomizations-policy: pre-validation: couldn't find model
, err: invalid kind" "action"="mutate" "gvk"="kustomize.toolkit.fluxcd.io/v1beta2, Kind=Kustomization" "operation"="UPDATE" "resource"="fluxcd/Kustomization/00-common"                             
 kyverno E0211 15:23:02.009443       1 mutation.go:87] WebhookServer "msg"="mutate error" "error"="failed to validate resource mutated by policy root-kustomizations-policy: pre-validation: couldn't find model
│  , err: invalid kind" "action"="mutate" "gvk"="kustomize.toolkit.fluxcd.io/v1beta2, Kind=Kustomization" "operation"="UPDATE" "resource"="fluxcd/Kustomization/00-common"                                        │                                                                                                                                                                                                              
                                                                                                                                                                                                                
``` 

### References
- https://github.com/kyverno/kyverno/issues/2094
### Affected releases

* 53.0.0

<!--
### Traces and logs

Remember this is a public repo. DON'T leak credentials or Orange internal URLs. 
Automation may be applied in the future. 

* [ ] I have reviewed provided traces against secrets (credentials, internal URLs) that should not be leake, manually of using some tools such as [truffle-hog file:///user/dxa4481/codeprojects/mytraces.txt](https://github.com/dxa4481/truffleHog#truffle-hog)
 -->

**Comment by poblin-orange** [comment]
workaround: kill pod in namespace kyverno (should let kyverno apply bridge policy)

**Comment by gberche-orange** [comment]
A possible fail-safe syntax would be to leave a `spec: {}` entry in the top Kustomization, so that a failure to apply the kyverno patch would leave an invalid Kustomization that would be rejected by the api server.

**Comment by JCL38-ORANGE** [comment]
not reproduced during v530.1 upgrade with 00-marketplace and 00-supervision from scratch

**Comment by gberche-orange** [comment]
Reproduced during v54-rc build phases on cluster gitops-management

**Comment by gberche-orange** [comment]
Seems fixed in kyverno 1.5.0
We still use `helm_kyverno: "v2.0.3"` running `ghcr.io/kyverno/kyverno:v1.4.3`  where as 2.3.0 is the latest on main branch
https://github.com/kyverno/kyverno/blob/main/charts/kyverno/Chart.yaml#L4-L5

```
version: v2.3.0
appVersion: v1.6.0
```

curl https://kyverno.github.io/kyverno/index.yaml also lists 2.3.1 version
